operational_analysis/methods/plant_analysis.py

```diff
@@ -335,10 +348,11 @@ def process_revenue_meter_energy(self):
         # Get actual number of days per month in the raw data
         # (used when trimming beginning and end of monthly data frame)
         # If meter data has higher resolution than monthly
-        if (self._plant._meter_freq != '1MS') & (self._plant._meter_freq != '1M'):
-            self._monthly.df['num_days_actual'] = df.resample('MS')['energy_kwh'].apply(tm.num_days)
+        if (self._plant._meter_freq == '1MS') | (self._plant._meter_freq == '1M'):
```

Comment:
Didn't like how this was set up (similar to dissatisfaction with range_flag), so I adjusted. Same effect.